Source/WebKitLegacy/mac/WebView/WebHTMLView.mm

```diff
@@ -6153,10 +6153,7 @@ - (void)attachRootLayer:(CALayer *)layer
         // If we are in a layer-backed view, we need to manually initialize the geometry for our layer.
         [viewLayer setBounds:NSRectToCGRect([_private->layerHostingView bounds])];
         [viewLayer setAnchorPoint:CGPointMake(0, [self isFlipped] ? 1 : 0)];
-        ALLOW_DEPRECATED_DECLARATIONS_BEGIN
-        CGPoint layerPosition = NSPointToCGPoint([self convertPointToBase:[_private->layerHostingView frame].origin]);
-        ALLOW_DEPRECATED_DECLARATIONS_END
-        [viewLayer setPosition:layerPosition];
+        [viewLayer setPosition:NSPointToCGPoint([self convertPointToBacking:[_private->layerHostingView frame].origin])];
     }
```

Comment:
Have you tested this? I'm not an AppKit expert, but it's not clear to me from API documentation that this is a change that's safe to make mechanically.

Reply:
I absolutely have.

Reply:
Marking unresolved for sign-off from a review with more AppKit knowledge than me.

Reply:
It's not apparent that this change is safe to make. `convertPointToBase` and `convertPointToBacking` do not have exactly equal implementations.

I would also err on the side of caution when removing code in WebKitLegacy, since all code here is effectively deprecated.

@smfr do you recall why resolution independence is needed here? I wonder if this was really intended to be in window coordinates?

Reply:
I do not recall. But I would prefer this PR be broken into one for each part.